airbyte-integrations/bases/base-normalization/integration_tests/test_ephemeral.py

```diff
@@ -28,8 +28,7 @@ def before_all_tests(request):
         "test_type": "ephemeral",
         "tmp_folders": temporary_folders,
     }
-    if DestinationType.POSTGRES.value not in destinations_to_test:
-        destinations_to_test.append(DestinationType.POSTGRES.value)
+
```

Comment:
This change is because I branch out of master for this PR instead of my branch. I don't want to rebase my other PR with master because they are already green and ready to merge. Please ignore this change when reviewing.